mitodl/open-discussions
# Added base_url to password reset email

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1289 

#### What's this PR do?
Fixes a regression on password reset emails

#### How should this be manually tested?
Ensure you have mailgun configured, send yourself a password reset email and verify it works
